package.json

```diff
@@ -4,9 +4,13 @@
   "description": "AsyncAPI schema versions",
   "main": "index.js",
   "scripts": {
-    "test": "nyc mocha",
+    "test": "npm run build && nyc mocha",
+    "build": "npm run bundle",
     "release": "semantic-release",
-    "generate:assets": "echo 'No additional assets need to be generated at the moment'",
+    "generate:assets": "npm run build",
+    "prepublishOnly": "npm run build",
+    "bundle": "cd tools/bundler && npm i && npm run bundle",
+    "startNewVersion": "newVersion=$npm_config_new_version && latestVersion=$(ls -d ./definitions/* | sort -V -r | head -1 | xargs -n 1 basename) && [ -d ./definitions/${newVersion} ] && echo Directory ./definitions/$newVersion already exist and cannot be overwritten. Please create a different version. || (cp -R ./definitions/$latestVersion ./definitions/$newVersion && find ./definitions/$newVersion -name '*.json' -exec sed -i '' \"s+definitions/$latestVersion+definitions/$newVersion+g\" {} +)",
```

Comment:
😱 😆

Reply:
Yeea, not sure if we should just leave it, or create something like a `script` folder? 🤔

Reply:
I definitely recommend moving it out into separate script, it is scary, and in separate script you can add comments 😅

Reply:
I'll make sure to split it out in another PR to simplify it 🙂